library/alloc/src/vec/mod.rs

```diff
@@ -2338,7 +2338,7 @@ impl<T: Clone, A: Allocator + Clone> Clone for Vec<T, A> {
     #[cfg(not(test))]
     fn clone(&self) -> Self {
         let alloc = self.allocator().clone();
-        <[T]>::to_vec_in(&**self, alloc)
+        self.to_vec_in(alloc)
```

Comment:
I don't think this actually changes anything? This is still getting auto-deref'd to the same method on `[T]`.

Reply:
Yes, it's the same method, just seemed more readable